wildfly-users-ejb/src/main/java/gallb/wildfly/users/ejb/userManagement/business/UserBean.java

```diff
@@ -0,0 +1,110 @@
+package gallb.wildfly.users.ejb.userManagement.business;
+
+import java.util.List;
+
+import javax.ejb.Stateless;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.PersistenceException;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Root;
+
+import org.jboss.logging.Logger;
+
+import gallb.wildfly.users.common.IUser;
+import gallb.wildfly.users.ejb.exception.EjbException;
+import model.User;
+
+
+
+@Stateless
+public class UserBean implements IUser {
+	@PersistenceContext(unitName = "WildflyUsers")
+	private EntityManager oEntityManager;
+	private Logger oLogger = Logger.getLogger(UserBean.class);
+
+	@Override
+	public List<User> getAll() throws EjbException {
+		try {
+			@SuppressWarnings("unchecked")
+			List<User> users = (List<User>) oEntityManager.createNamedQuery("User.findAll").getResultList();
+			return users;
+		} catch (PersistenceException e) {
+			oLogger.error(e);
+			EjbException.getCause(e);
+			throw new EjbException("Can't find users", e);
+		}
+	}
+
+	@Override
+	public User getById(int id) throws EjbException {
+		try {
+			User u = oEntityManager.find(User.class, id);

```

Comment:
itt is lehet egybol visszateriteni, nincs ertelme behozni egy uj lokalis valtozot: "User u"